jena-arq/src/main/java/org/apache/jena/sparql/expr/nodevalue/NodeFunctions.java

```diff
@@ -254,28 +254,33 @@ public static String lang(Node node) {
     }
 
     // -------- langMatches
+    /** LANGMATCHES - first argument is a language string */ 
```

Comment:
This seems like a `@param`?
```
/** LANGMATCHES 
      @param nv a language string
*/ 
```

Reply:
It is a reminder that LANGMATCHEs, in SPARQL takes a language string, not the literal with the language string, which catches people out sometime. I can turn it into \@params.

Reply:
Ah, I didn't know that, but at least I just learnt about `LANGMATCHES`!